projects/demo/src/app/app.component.ts

```diff
@@ -1,6 +1,6 @@
 import {ChangeDetectionStrategy, Component} from '@angular/core';
 import {DemoPath} from '@demo/constants';
-import {TUI_SANITIZER} from '@taiga-ui/core';
+import {TUI_SANITIZER} from '@taiga-ui/legacy';
```

Comment:
We do not need this anymore at all

Reply:
Deleted in https://github.com/taiga-family/maskito/pull/1390/commits/6868980272983af084bcbf3a522945d95e3b4dd5